runelite-client/src/main/resources/net/runelite/client/plugins/chatcommands/pets.json

```diff
@@ -0,0 +1,194 @@
+[
+  {
```

Comment:
This would probably be better represented just by having a `Pet` enum with a `(name, item)` constructor. Then you can just use `Pet.values()` to get a `Pet[]`